ource

```diff
@@ -0,0 +1,52 @@
+[1mdiff --git a/source/speech/speech.py b/source/speech/speech.py[m
```

Comment:
something looks wrong with this file - it looks like piped output from a diff - should it exist?

Reply:
Fixed in 6a3f2c6db1e07d7ec3041d79110afdd97eea3090